### PR TITLE
Userstory 2

### PR DESCRIPTION
**Pull request by Malte1983**
add Footer.js as a form. 
add Footer.spec and test.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -31,10 +32,24 @@ function App() {
...
+    const newQuestion = [
+      ...data,
+      {
+        id: i,
+        text: text,
+        author: author,
+      },
+    ]
+    setData(newQuestion)
+  }
```
Comment (StefanBehring) [inline client/src/App.js]: this function won't work as expected once you reach user story 4. 
if you have for example 3 items in your data array and delete the second one you will have 2 left. the last one then has the id 3. if you then add a new question the id of the new question will be 3, too
hint: nanoid

File: client/src/components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components/macro'
+import React from 'react'
+
+function Footer({ onCreateQuestion }) {
+  return (
+    <Wrapper onSubmit={e => handleSubmit(e)}>
+      <Label aria-label="Add new question">
```
Comment (StefanBehring) [inline client/src/components/Footer/Footer.js:7]: Nice! :)

**Review by StefanBehring — COMMENTED** [review]
Well done!

File: client/src/App.js (hunk 1 above)
Comment (mbosselmann) [reply client/src/App.js]: Thank you! We switched to nanoid! :-)

File: client/src/components/Footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        <TextInput
+          name="question"
+          placeholder="Enter your question here!"
+          type="text"
+        />
+      </Label>
+      <SubmitButton aria-label="Submit question">Submit</SubmitButton>
```
Comment (macfrei) [inline client/src/components/Footer/Footer.js:14]: The `aria-label` here is not super necessary as `Submit` should be sufficient enough for users to understand. Now screen reader user will get `Submit question` and everyone else `Submit`.

**Pull request merged**